app.models/api.models.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection");
+const fs = require("fs/promises");
+
+exports.fetchApi = () => {
+  return fs.readFile(`${__dirname}/../endpoints.json`, "utf8").then((data) => {
+    return data
+  });
+};
```

Comment:
Yes good work, using `.readFile` is perfectly fine to do here.

Reply:
Thank you!